specification/conformance/conformance_rules/columns/tags.json

```
@@ -0,0 +1,427 @@
+                    },
+                    {
+                        "CheckFunction": "CheckConformanceRule",
+                        "ConformanceRuleId": "Tags-C-009-M"
+                    },
+                    {
+                        "CheckFunction": "CheckConformanceRule",
+                        "ConformanceRuleId": "Tags-C-010-C"
+                    }
+                ]
+            },
+            "Condition": {
+                "CheckFunction": "CheckValue",
+                "ColumnName": "Tags",
+                "Value": null
+            },
+            "Dependencies": [
+                "Tags-C-004-M",
+                "Tags-C-005-M",
+                "Tags-C-006-M",
+                "Tags-C-007-O",
+                "Tags-C-008-M",
+                "Tags-C-009-M",
+                "Tags-C-010-C"
+            ]
+        }
+    },
+    "Tags-C-004-M": {
+        "Function": "Validation",
+        "Reference": "Tags",
+        "EntityType": "Column",
+        "Notes": "",
+        "CRVersionIntroduced": "1.2",
+        "Status": "Active",
+        "ApplicabilityCriteria": [],
+        "Type": "Dynamic",
+        "ValidationCriteria": {
+            "MustSatisfy": "Tags MUST include all user-defined and provider-defined tags.",
+            "Keyword": "MUST",
+            "Requirement": {},
+            "Condition": {},
+            "Dependencies": []
+        }
+    },
+    "Tags-C-005-M": {
+        "Function": "Validation",
+        "Reference": "Tags",
+        "EntityType": "Column",
+        "Notes": "",
+        "CRVersionIntroduced": "1.2",
+        "Status": "Active",
+        "ApplicabilityCriteria": [],
+        "Type": "Dynamic",
+        "ValidationCriteria": {
+            "MustSatisfy": "Tags MUST only include finalized tags.",
+            "Keyword": "MUST",
+            "Requirement": {},
+            "Condition": {},
+            "Dependencies": []
+        }
+    },
+    "Tags-C-006-M": {
+        "Function": "Validation",
+        "Reference": "Tags",
+        "EntityType": "Column",
+        "Notes": "",
+        "CRVersionIntroduced": "1.2",
+        "Status": "Active",
+        "ApplicabilityCriteria": [],
+        "Type": "Dynamic",
+        "ValidationCriteria": {
+            "MustSatisfy": "Tags SHOULD include tag keys with corresponding non-null values for a given resource.",
+            "Keyword": "SHOULD",
+            "Requirement": {},
+            "Condition": {},
+            "Dependencies": []
+        }
+    },
+    "Tags-C-007-O": {
+        "Function": "Nullability",
+        "Reference": "Tags",
+        "EntityType": "Column",
+        "Notes": "",
+        "CRVersionIntroduced": "1.2",
+        "Status": "Active",
+        "ApplicabilityCriteria": [],
+        "Type": "Dynamic",
+        "ValidationCriteria": {
+            "MustSatisfy": "Tags MAY include tag keys with a null value for a given resource depending on the provider’s tag finalization process.",
```

Editorial nit: replaced smart quote (’) with straight quote (') for consistency

```suggestion
            "MustSatisfy": "Tags MAY include tag keys with a null value for a given resource depending on the provider's tag finalization process.",
```